src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -3152,13 +3152,17 @@ object Desugar {
 
     private def name(postfix: String)(n: String, s: PScope, context: ExternalTypeInfo): String = {
       maybeRegister(s, context)
+      val pkgId = context.getTypeInfo.tree.originalRoot.viperId
```

Comment:
please extend `ExternalTypeInfo` to have a `packageInfo` getter (similar to `pkgName`)